api/v1beta1/common_types.go

```diff
@@ -37,6 +46,10 @@ type WatcherCommon struct {
 	// +kubebuilder:default=watcher
 	// DatabaseAccount - MariaDBAccount CR name used for watcher DB, defaults to watcher
 	DatabaseAccount string `json:"databaseAccount"`
+
+	// +kubebuilder:validation:Optional
+	// The service specific Container Image URL (will be set to environmental default if empty)
+	ContainerImage string `json:"containerImage"`
```

Comment:
WatcherCommon is used in all CRDs, included Watcher, but Watcher should not have ContainerImage, but WatcherImages. ContainerImage should go out of WatcherCommon, we may create another struct for the params that go to the 2nd level CRs but not to Watcher, or add it to them directly.

Reply:
Done! Added  [WatcherContainerImages](https://github.com/raukadah/watcher-operator/blob/webhook_default_images/api/v1beta1/common_types.go#L73) struct to achieve the same.